Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceSolver.H

```diff
@@ -0,0 +1,56 @@
+#ifndef WARPX_FINITE_DIFFERENCE_SOLVER_H_
+#define WARPX_FINITE_DIFFERENCE_SOLVER_H_
+
+#include <AMReX_MultiFab.H>
```

Comment:
Pls move this include *after* the WarpX-local includes.

Reply:
just a ping, this was probably forgotten

Reply:
There was some refactoring, so now the WarpX-local includes are not used anymore in this file.